grafana-init/grafana.py

```diff
@@ -30,14 +30,14 @@
 GRAFANA_URL = os.environ.get('GRAFANA_URL', 'http://grafana:3000')
 GRAFANA_USERNAME = os.environ.get('GRAFANA_USERNAME', 'mini-mon')
 GRAFANA_PASSWORD = os.environ.get('GRAFANA_PASSWORD', 'password')
+GRAFANA_USERS = [{'user': GRAFANA_USERNAME, 'password': GRAFANA_PASSWORD, 'email': ''}]
 
 DATASOURCE_NAME = os.environ.get('DATASOURCE_NAME', 'monasca')
 DATASOURCE_URL = os.environ.get('DATASOURCE_URL', 'http://monasca:8070/')
 DATASOURCE_ACCESS_MODE = os.environ.get('DATASOURCE_ACCESS_MODE', 'proxy')
 
 DASHBOARDS_DIR = os.environ.get('DASHBOARDS_DIR', '/dashboards.d')
 
-
```

Comment:
Don't remove this empty line, flake8 (linter) will complain that before every `def` (or `@retry` before `def`) we need two empty lines.

Reply:
Sorry for late update.
Done.